components/Tables/ContributorsTable.tsx

```diff
@@ -252,7 +254,7 @@ export default function ContributorsTable({
   const table = useReactTable({
     columns: useMemo(
       () => (isMobile ? mobileColumns({ isLoggedIn }) : defaultColumns({ repository, isLoggedIn })),
-      [isMobile]
+      [isMobile, isLoggedIn]
```

Comment:
Adding `isLoggedIn` was required in the dependency array so that the UI updates to show or hide the OSCR.